site/content/docs/single-sign-on/saml.md

```diff
@@ -18,13 +18,26 @@ Setting up SSO for your organisation starts with providing [Checkly Support](mai
 - sign out URL
 - public X509 certificate
 
-## Testing the SSO integration
+## IdP configuration
 
 After configuration has taken place on Checkly's side, you will receive the following information to configure your IdP:
 
-- a redirect URL
-- a signout URL
-- configuration details in XML format
+- a redirect URL (e.g. `https://auth.checklyhq.com/login/callback?connection=<YOUR_CONNECTION_ID>`)
+- a signout URL (normally `https://auth.checklyhq.com/logout`)
+- XML metadata (e.g. `https://auth.checklyhq.com/samlp/metadata?connection=<YOUR_CONNECTION_ID>`)
+
+Please make sure that Checkly is sent the email address back from your SSO Idp. This way Checkly can map the existing Checkly user to your Idp user. The object returned should have an “email” field, e.g.:
```

Comment:
I'm not 100% sure about the part below. That is why I wanna research about it this week

Reply:
@tnolet could you please approve the PR? I will then merge.